nodejs/nodejs.dev
# Fix same key bug

## Description


This PR updates the `key` prop for the `section` element in the `docs` page. This was required as `page.name` was **not unique** within its children.

How to test?
1. Checkout the branch.
2. Run the server and route to `/docs` page.
3. Open up the dev console. You **should not** see any warning related to React keys **for any of the sections**  

## Related Issues



[Issue Link](https://github.com/nodejs/nodejs.dev/issues/956)
